src/components/Chart/Chart.jsx

```diff
@@ -21,7 +21,7 @@ import { bool } from 'prop-types';
 
 import { indexToTimestamp, options, timestampToIndex } from '../../globals';
 import { useLazyInitializedRef } from '../../hooks/useLazyInitializedRef';
-import {
+import chartReducer, {
```

Comment:
Blir denne brukt?

Reply:
It might be that it thinks importing it runs some code necessary to set things up I guess.